vscode-languageserver-types/src/lib.rs.in

```diff
@@ -133,6 +133,7 @@ pub struct DidChangeWatchedFilesParams {
 /**
  * The file event type.
  */
```

Comment:
It appears as doc comments are not normalized (so `/* */` are normalized but not `/** */`). There is not option for doc comments as well...